mmv1/third_party/terraform/services/compute/resource_compute_network_firewall_policy_rule_test.go.tmpl

```diff
@@ -149,6 +147,185 @@ func TestAccComputeNetworkFirewallPolicyRule_securityProfileGroup_update(t *test
+  capacity    = 100
+}
+
+resource "google_compute_network_firewall_policy" "basic_network_firewall_policy" {
+  name        = "tf-test-policy-%{random_suffix}"
+  description = "Sample global network firewall policy"
+  project     = "%{project_name}"
+}
+
+resource "google_compute_network_firewall_policy_rule" "primary" {
+  action                  = "allow"
+  description             = "This is a simple rule description"
+  direction               = "INGRESS"
+  disabled                = false
+  enable_logging          = true
+  firewall_policy         = google_compute_network_firewall_policy.basic_network_firewall_policy.id
+  priority                = 1000
+  tls_inspect             = false
+  rule_name               = "tf-test-rule-%{random_suffix}"
+
+  match {
+    src_ip_ranges = ["10.100.0.1/32"]
+    src_fqdns = ["google.com"]
+    src_region_codes = ["US"]
+    src_threat_intelligences = ["iplist-known-malicious-ips"]
+
+    src_secure_tags {
+      name = "tagValues/${google_tags_tag_value.basic_value.name}"
+    }
+
+    layer4_configs {
+      ip_protocol = "all"
+    }
+    
+    src_address_groups = [google_network_security_address_group.basic_global_networksecurity_address_group.id]
+  }
+}
+
+resource "google_compute_network" "basic_network" {
+  name = "tf-test-network-%{random_suffix}"
+  auto_create_subnetworks = false
+}
+
+resource "google_tags_tag_key" "basic_key" {
+  description = "For keyname resources."
+  parent      = "organizations/%{org_id}"
+  purpose     = "GCE_FIREWALL"
+  short_name  = "tf-test-tagkey-%{random_suffix}"
+  purpose_data = {
+    network = "%{project_name}/${google_compute_network.basic_network.name}"
+  }
+}
+
+resource "google_tags_tag_value" "basic_value" {
+  description = "For valuename resources."
+  parent      = "tagKeys/${google_tags_tag_key.basic_key.name}"
+  short_name  = "tf-test-tagvalue-%{random_suffix}"
+}
+`, context)
+}
+
+func testAccComputeNetworkFirewallPolicyRule_secureTagsUpdate(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_network_security_address_group" "basic_global_networksecurity_address_group" {
+  name        = "tf-test-address-%{random_suffix}"
+  parent      = "projects/%{project_name}"
+  description = "Sample global networksecurity_address_group"
+  location    = "global"
+  items       = ["208.80.154.224/32"]
+  type        = "IPV4"
+  capacity    = 100
+}
+
+resource "google_compute_network_firewall_policy" "basic_network_firewall_policy" {
+  name        = "tf-test-policy-%{random_suffix}"
+  description = "Sample global network firewall policy"
+  project     = "%{project_name}"
+}
+
+resource "google_compute_network_firewall_policy_rule" "primary" {
+  action          = "deny"
+  description     = "This is an updated rule description"
+  direction       = "EGRESS"
+  disabled        = true
+  enable_logging  = false
+  firewall_policy = google_compute_network_firewall_policy.basic_network_firewall_policy.id
```

Comment:
```suggestion
  firewall_policy = google_compute_network_firewall_policy.basic_network_firewall_policy.id
  project              = "%{project_name}"
```

And this would update to the short-form project name, (hopefully) proving that the diff suppress function removed by the migration was unnecessary.